package.json

```diff
@@ -8,7 +8,7 @@
     "@wireapp/avs": "9.7.15",
     "@wireapp/commons": "5.2.8",
     "@wireapp/core": "46.0.10",
-    "@wireapp/react-ui-kit": "9.17.5",
+    "@wireapp/react-ui-kit": "9.17.6",
```

Comment:
The CircleCloseIcon issue is fixed on the ui-kit side, see https://github.com/wireapp/wire-web-packages/pull/6264